jvm/src/test/scala/org/scalacheck/GenSpecification.scala

```diff
@@ -124,6 +124,13 @@ object GenSpecification extends Properties("Gen") with GenSpecificationVersionSp
     }
   }
 
+  property("choose-big-int") = forAll { (l: BigInt, h: BigInt) =>
+    Try(choose(l, h)) match {
+      case Success(g) => forAll(g) { l <= x && x <= h }
```

Comment:
This fails to compile: `not found: value x`

Reply:
Thanks, I think I didn't have enough coffee yesterday.